src/vendors/ng2-page-scroll/ng2-page-scroll.ts

```diff
@@ -0,0 +1,7 @@
+export * from './src/ng2-page-scroll.directive';
```

Comment:
if aot is not supported in ng2-page-scroll maybe make sense to find alternative, or make PR to ng2-p-s